src/plugins/data_views/server/rest_api_routes/public/get_data_views.ts

```diff
@@ -66,6 +66,12 @@ const getDataViewsRouteFactory =
     router.versioned.get({ path, access: 'public', description }).addVersion(
       {
         version: INITIAL_REST_VERSION,
+        security: {
+          authz: {
+            enabled: false,
+            reason: 'Authorization provided by saved objects client',
```

Comment:
I'm curious why we use `indexPatterns:read` for getting a single data view, but not the list?

Reply:
Yeah, that was not intentional.